lib/uplink/bucket_attrs_test.go

```diff
@@ -57,6 +58,55 @@ func simpleEncryptionAccess(encKey string) (access EncryptionAccess) {
 	return access
 }
 
+func createPartnerID() (partnerID string, err error) {
+	id, err := uuid.New()
+	if err != nil {
+		return partnerID, err
+	}
+	return id.String(), err
+}
+
+// check that partner bucket attributes are stored and retrieved correctly.
+func TestPartnerBucketAttrs(t *testing.T) {
+	var (
+		access     = simpleEncryptionAccess("voxmachina")
+		bucketName = "mightynein"
+	)
+
+	testPlanetWithLibUplink(t, testConfig{},
+		func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, proj *Project) {
+			_, err := proj.CreateBucket(ctx, bucketName, nil)
+			require.NoError(t, err)
+
+			partnerID, err := createPartnerID()
```

Comment:
There is `testrand.UUID().String()` now.

Reply:
updated the code to use testrand